tests/modeladmin/test_checks.py

```diff
@@ -1102,6 +1156,20 @@ class TestModelAdmin(ModelAdmin):
             invalid_obj=ValidationTestInline
         )
 
+        class ValidationTestInlineWithoutFormsetClass(TabularInline):
```

Comment:
I think can be a separate test method, it doesn't seem to depend on anything above.

Reply:
Since there were only two methods, one for invalid test and another for a valid test, I thought to add this in the invalid test method only. I will separate it out in another method.